src/components/Bar.jsx

```diff
@@ -117,19 +121,29 @@ class Bar extends Component {
   render () {
     const { t } = this.props
     const { fireClaudy, displaySupport, enableSearchBar } = this.state
+    const { barLeft, barRight, barCenter } = this.props
     return (
       <div className='coz-bar-container'>
-        { this.renderLeft() }
-        { this.renderCenter() }
+        { barLeft || this.renderLeft() }
+        { barCenter || this.renderCenter() }
         { enableSearchBar
           ? <SearchBar />
           : <hr className='coz-sep-flex' key='separator'/>
         }
-        { this.renderRight() }
+        { barRight || this.renderRight() }
         {displaySupport && <SupportModal onClose={this.toggleSupport} />}
       </div>
     )
   }
 }
 
-export default translate()(Bar)
+const mapStateToProps = state => ({
+  barLeft: getContent(state, 'left'),
+  barRight: getContent(state, 'right'),
+  barCenter: getContent(state, 'center')
+})
+
+export default compose(
```

Comment:
Why using `compose` (and its dependency) here? Can't we do:
```js
expose default translate()(connect(mapStateToProps, null)(Bar))
```

Reply:
I find the `compose` version more readable.

Reply:
I do not think there is a problem with having `lodash` as a dependency.

Reply:
No problem with `compose` from `lodash`, but since we don't use it anywhere else for now (and we can do it without), I would prefer without, thanks 👍

Reply:
I agree with @CPatchane, I am not a big fan of dependencies, especially lodash. I think we should not add a dependency for just one "comfort function".

However, if we really want to use `compose` for exporting our components, we should maintain code coherence and use it everywhere. And in every app.

Reply:
It is not really adding a dependency since you have already 32 dependencies that depends on it.

Why are you not a fan of lodash ?

Reply:
I personally think that it's just a lib of shortcuts hiding intersting JS behavior and patterns.

But I am not against using it. But if we do so, it should be done in all our components in every app.